doc/user_guide/distributed_big_data.rst

```diff
@@ -0,0 +1,75 @@
+.. _Big Data:
+
+Big Data
+---------
+
+4-D STEM datasets are large and difficult to work with.  In ``pyxem`` we get around this by
+using a lazy loading approach.  This means that the data is not loaded into memory until it is
+needed.
+
+This is a very powerful approach, but it can be confusing at first. In this guide we will
+discuss how to work with large datasets in ``pyxem``.
+
+.. note::
+    If you want more information on working with large datasets in ``pyxem``, please see the
+    `Big Data<https://hyperspy.org/hyperspy-doc/current/user_guide/big_data.html>_` section of
+    the `HyperSpy User Guide<https://hyperspy.org/hyperspy-doc/current/user_guide/index.html>`_.
+
+Loading and Plotting a Dataset
+------------------------------
+
+Let's start by loading a dataset.  We will use the ``load`` function to load a dataset from
+hyperspy
+
+.. code-block:: python
+
+    >>> import hyperspy.api as hs
+    >>> s = hs.load("big_data.zspy", lazy=True)
+    >>> s
+
+The dataset here is not loaded into memory here so this should happen instantaneously. We can
+then plot the dataset.
+
+.. code-block:: python
+
+    >>> s.plot()
+
+Which (in general) will be very slow.  This is because the data is being loaded into memory and a
+navigator image is being generated before the plot is shown. In many cases a HAAADF dataset will be
+collected as well as a 4-D STEM dataset.  In this case we can set the navigator to the HAAADF dataset instead.
+
+.. code-block:: python
+    >>> haadf = hs.load("haadf.zspy") # load the HAAADF dataset
+    >>> s.naviagtor = haadf # set the navigator to the HAAADF dataset
+    >>> s.plot() # happens instantaneously
```

Comment:
@hakonanes The idea is that if you save the data the navigator is saved as well (Under .metedata._Hyperspy.navigator). I like to do this mostly because I __hate__ when I accidently call `s.plot()` without a navigator object and it tries to compute the navigator for some very large dataset.